deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -130,9 +135,19 @@ Resources:
               - 's3:PutBucketAcl'
               - 's3:GetBucketAcl'
               - 's3:PutBucketWebsite'
+              - 's3:DeleteBucketWebsite'
+              - 's3:DeleteBucketPolicy'
+              - 's3:PutBucketPolicy'
+              - 's3:GetBucketPolicy'
             Resource:
               - !Sub 'arn:aws:s3:::${TestStackName}*'
               - !Sub 'arn:aws:s3:::${ProdStackName}*'
+              - Fn::Join:
+                - ""
+                -
+                  - 'arn:aws:s3:::'
+                  - Fn::ImportValue: !Join [':', [!Ref InfrastructureStackName, 'LogBucket']]
+                  - '*'
```

Comment:
Does the Import have the trailing slash? You might need to change L150 to `- '/*'` .

Reply:
the import does not have a trailing slash.

Reply:
If you've checked that this policy works the way you expect, I'm fine here. I am just unsure if it will function.